src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous"
+    >
```

Comment:
You need to check and fix it everywhere
<img width="1029" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/0bd82a7c-9cb3-4de0-85c2-beb5da94bd43">

Reply:
Tried to resolve it everywhere in HTML, where I could find it